server/poll/poll.go

```diff
@@ -355,8 +355,13 @@ func (s Settings) String() string {
 	if s.PublicAddOption {
 		settingsText = append(settingsText, "public-add-option")
 	}
-	if s.MaxVotes > 1 {
-		settingsText = append(settingsText, fmt.Sprintf("votes=%d", s.MaxVotes))
+	if s.MaxVotes != 1 {
+		switch s.MaxVotes {
+		case 0:
+			settingsText = append(settingsText, "votes=all")
+		default:
+			settingsText = append(settingsText, fmt.Sprintf("votes=%d", s.MaxVotes))
```

Comment:
For all settings? If so, I don't feel that just translating it would be very effective, so if you are going to do it, you may need to revamp this part of the UI as a whole. 0/5
<img width="728" alt="image" src="https://github.com/user-attachments/assets/c75c5205-5951-4766-afff-c5d9a0073a77" />